addons/hr_attendance/hr_attendance.py

```diff
@@ -51,7 +51,7 @@ def _worked_hours_compute(self, cr, uid, ids, fieldnames, args, context=None):
                     last_signin_datetime = datetime.strptime(last_signin.name, '%Y-%m-%d %H:%M:%S')
                     signout_datetime = datetime.strptime(obj.name, '%Y-%m-%d %H:%M:%S')
                     workedhours_datetime = (signout_datetime - last_signin_datetime)
-                    res[obj.id] = ((workedhours_datetime.seconds) / 60) / 60.0
+                    res[obj.id] = ((workedhours_datetime.total_seconds()) / 60) / 60.0
```

Comment:
Hello Stefan, we use that code to calculate attendance more than 24h (people that stay more than 24h at the same place)